support-frontend/assets/components/forms/customFields/options.tsx

```diff
@@ -1,13 +1,17 @@
 // ----- Imports ----- //
 import { Option } from '@guardian/src-select';
-import type { Node } from 'react';
 import React from 'react';
 
 // ----- Functions ----- //
-const options = (optionsForMapping: Record<string, string>): Node =>
-	Object.keys(optionsForMapping).map((key) => (
-		<Option value={key}>{optionsForMapping[key]}</Option>
-	));
+const options = (optionsForMapping: Record<string, string>): JSX.Element => {
+	return (
+		<>
+			{Object.keys(optionsForMapping).map((key) => (
+				<Option value={key}>{optionsForMapping[key]}</Option>
+			))}
+		</>
+	);
+};
```

Comment:
Maybe not worth changing, but I'd be tempted to change this into a proper component